src/tests/index.test.js

```diff
@@ -67,10 +92,22 @@ describe('asset manager handles correctly', () => {
 			});
 	
 			it('should not confirm', () => {
+				const invokedJavascript = [];
+				global.window = {
```

Comment:
Mocking `window` is tricky. You generally want to save off the original the original value of `window` into a temp variable before redefining it, doing so in a `beforeEach()`. Then, using `afterEach()`, restore the original value so as to not affect the assumptions of subsequent tests.

Reply:
Updated to replace window